waspc/src/Wasp/Generator/DbGenerator/Common.hs

```diff
@@ -60,3 +66,42 @@ dbSchemaChecksumOnLastGenerateFileInDbRootDir = [relfile|schema.prisma.wasp-gene
 
 dbSchemaChecksumOnLastGenerateFileProjectRootDir :: Path' (Rel ProjectRootDir) (File DbSchemaChecksumOnLastGenerateFile)
 dbSchemaChecksumOnLastGenerateFileProjectRootDir = dbRootDirInProjectRootDir </> dbSchemaChecksumOnLastGenerateFileInDbRootDir
+
+data MigrateArgs = MigrateArgs
+  { _migrationName :: Maybe String,
+    _isCreateOnlyMigration :: Bool
+  }
+  deriving (Show, Eq)
+
+emptyMigrateArgs :: MigrateArgs
+emptyMigrateArgs = MigrateArgs {_migrationName = Nothing, _isCreateOnlyMigration = False}
+
+asArgs :: MigrateArgs -> [String]
+asArgs migrateArgs = do
+  concat . concat $
+    [ [["--create-only"] | _isCreateOnlyMigration migrateArgs],
+      [["--name", name] | Just name <- [_migrationName migrateArgs]]
+    ]
+
+parseMigrateArgs :: Maybe [String] -> MigrateArgs
+parseMigrateArgs Nothing = emptyMigrateArgs
+parseMigrateArgs (Just migrateArgs) = do
+  go migrateArgs emptyMigrateArgs
+  where
+    go :: [String] -> MigrateArgs -> MigrateArgs
+    go ("--create-only" : rest) mArgs = go rest mArgs {_isCreateOnlyMigration = True}
+    go ("--name" : name : rest) mArgs = go rest mArgs {_migrationName = Just name}
+    go _ mArgs = mArgs
+
+data OnLastDbConcurrenceChecksumAction
```

Comment:
What exactly does this stand for? So on it tells me what was done on the last DB concurrent checksum, or what should be done?

Reply:
Ah got it know, it should be read as "Write OnLastDbConcurrenceChecksum".
Hmmmmmm.
I don't have much better idea. Maybe a short comment saying that this says what would be done with the checksum file.

Maybe `WriteOnLastDbConcurrentChecksumFile` would be better? So adding File at the end?

Or maybe `Write_OnLastDbConcurrentChecksumFile`?

This `On` is what confuses me, but I guess we can't get rid of it easily as it become name that we use on many places?

Reply:
Good call on maybe changing the name and adding comments. The trick here is it isn't only just write, it could be remove, or nothing (no-op). But I can think more on it to try and clear up the confusion. And it is complicated by the fact that what it refers to is a tricky notion (the On thing)... 🤔

Reply:
Does this help improve things any?

```haskell
-- | This type tells us what we need to do with the DbSchemaChecksumOnLastDbConcurrenceFile.
data RefreshOnLastDbConcurrenceChecksumFile
  = WriteOnLastDbConcurrenceChecksumFile
  | RemoveOnLastDbConcurrenceChecksumFile
  | IgnoreOnLastDbConcurrenceChecksumFile
```

Reply:
I think it does! File makes you realize there is a noun there, I think that is enough, and comments clears it up.

Reply:
Awesome, thanks!